src/defaults.tsx

```diff
@@ -457,6 +460,22 @@ function addNotebookTour(
       }
     }
   });
+
+  // clean
+  notebookTour.finished.connect((_, data) => {
+    switch (data.step.target) {
+      case '#jp-property-inspector':
+        commands.execute('filebrowser:activate');
+        if (nbTracker) {
+            const current = nbTracker.currentWidget;
+            if (current) {
+              docManager?.deleteFile(current.context.path);
```

Comment:
Removing the notebook is dangerous as the user may not have an empty notebook when launching the tour.

Could you change the logic to keep track of the notebook modified (aka set the notebook variable at a outer scope) and of the added cell? Then use that information to remove the cell.

To remove the cell, I advice you to get the `activeCellIndex` from then `notebook` variable. Then you can reset it at the end of the tour. And finally you can call `NotebookActions.deleteCells(notebook)`.

Reply:
ok, i can do that but :)

I think it would be better to create a notebook (~ / nb-tour.ipynb) at the start of the tour and delete it at the end. this
- would avoid modifying an existing notebook
- would allow the tour to be launched even if no notebook is open

What do you think ?

Reply:
That could be nice indeed. So I let you update the PR in one of the two directions.

Reply:
I failed to create the notebook correctly.

I think 
`docManager?.createNew('nb-tour.ipynb', 'Notebook', {name: 'python3'});`
takes too much time.

The notebook is created but the tour stops. therefore, for the moment I just remove the cells